nexuse2e-webapp/src/main/frontend/src/app/message-table/message-table.component.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { ComponentFixture, TestBed } from "@angular/core/testing";
+
+import { MessageTableComponent } from "./message-table.component";
+import { By } from "@angular/platform-browser";
+import { MatTableModule } from "@angular/material/table";
+import { TranslateModule } from "@ngx-translate/core";
+import { MatCheckboxModule } from "@angular/material/checkbox";
+import { RouterTestingModule } from "@angular/router/testing";
+
+describe("MessageTableComponent", () => {
+  let component: MessageTableComponent;
+  let fixture: ComponentFixture<MessageTableComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [
+        MatTableModule,
+        MatCheckboxModule,
+        RouterTestingModule,
+        TranslateModule.forRoot(),
+      ],
+      declarations: [MessageTableComponent],
+    }).compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(MessageTableComponent);
+    component = fixture.componentInstance;
+    component.messages = [
+      {
+        messageId: "sdasdad-sadsadasda-asdsadada",
```

Comment:
Should we extract this sample messages to central file and use a spread operator here to change the needed fields? Currently we repeat it one multi places and having a spread operator will point more to the details that are important in this test.

Reply:
Okay I found the test data now. So why not use it here?

Reply:
Used test data everywhere